android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -1001,8 +987,17 @@ public void onClick(View v) {
                                         new Handler().postDelayed(new Runnable() {
                                             @Override
                                             public void run() {
-                                                mParentScrollView.scrollTo(0, 0);
-                                                mRecyclerView.scrollToPosition(0);
+                                                if (!ConfigurationUtils.isTablet(mActivity)
+                                                        && ConfigurationUtils.isLandscape(
+                                                                mActivity)) {
+                                                    mParentScrollView.smoothScrollTo(0, 0);
+                                                } else {
+                                                    mParentScrollView.smoothScrollTo(0,
+                                                            mRecyclerView.getHeight()
+                                                                    - mParentScrollView
+                                                                              .getMaxScrollAmount()
+                                                                    + dpToPx(getContext(), 90));
+                                                }
                                             }
                                         }, 300);
```

Comment:
@tapanmodh did you try with 100 here?maybe it can work the same way a be a bit faster. wdyt?

Reply:
good idea. Let me try with 100 and see.